scripts/deploy_oak_functions_endpoints

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+# shellcheck source=scripts/gcp_common
+source "$SCRIPTS_DIR/gcp_common"
+
+gcloud auth activate-service-account \
```

Comment:
I was not able to run this with the Cloud Run service account that we currently have, I had to use my personal account. 
Here is the first error message:
```
+ gcloud builds submit --tag gcr.io/oak-ci/endpoints-runtime-serverless:2.27.0-weather-lookup-endpoint-62sa4xcfia-nw.a.run.app-2021-06-11r7 . --project=oak-ci
ERROR: (gcloud.builds.submit) HTTPError 403: cloud-run@oak-ci.iam.gserviceaccount.com does not have storage.buckets.list access to the Google Cloud project.
```
We need a service account with additional permissions.

Reply:
This is fixed now.